sources/TileDB.CSharp/Enums.cs

```diff
@@ -652,6 +656,37 @@ public enum FilterOption : uint
         ScaleFloatByteWidth = tiledb_filter_option_t.TILEDB_SCALE_FLOAT_BYTEWIDTH,
         ScaleFloatFactor = tiledb_filter_option_t.TILEDB_SCALE_FLOAT_FACTOR,
         ScaleFloatOffset = tiledb_filter_option_t.TILEDB_SCALE_FLOAT_OFFSET,
+        /// <summary>
+        /// The WebP filter's compression quality.
+        /// </summary>
+        /// <remarks>
+        /// Must be a <see cref="float"/> between 1 and 100.
+        /// </remarks>
+        WebpQuality = tiledb_filter_option_t.TILEDB_WEBP_QUALITY,
+        /// <summary>
+        /// The WebP filter's input format.
+        /// </summary>
+        /// <remarks>Must be a <see cref="byte"/>. The allowed values are listed below:
```

Comment:
I will make use of `tiledb_filter_webp_format_t` after the strongly-typed `Filter` API.

Reply:
Changed my mind.